### ISSUE TITLE
AMD Support

### ISSUE THREAD

**Issue by sandikata** [cyberus-technology/virtualbox-kvm#26]
Hello,

as long as kvm support natively AMD processors, why is there still lacking this support?

Is it there any real reason for that (i don't think so as i am running many VMs through KVM), or any concerns from your side ?

Or Bridged network is not implemented yet at all?

Regards
Rosen Aleksandrov

**Comment by parthy** [comment]
We are currently in the process of validating AMD. We have not seen any issues yet, so it should be perfectly fine to give it a try. It's just not on the official support list at this point. If you test on AMD, feel free to report your hardware model and experience!

For bridged networking, we already have #25 and #3. That's independent of AMD.

**Comment by sandikata** [comment]
I am sure this is not an AMD issue, if we except that everything else is working as it works on bare KVM.

I am not sure about video adaptors, just haven't tested.

**Comment by tpressure** [comment]
@sandikata as @parthy already said, running the KVM backend on AMD should not be an issue. Do you see any problems on your side?

Bridged networking is neither supported on Intel nor on AMD. See #25

**Comment by sandikata** [comment]
No, haven't seen issues at all. But haven't gone too deep as there's no bridge which i am using intensive.

**Comment by fluoriteByte** [comment]
Can confirm Vbox-kvm is running fine-ish on my Amd ryzen 5 5500u laptop, except for some crashes on Windows ME and Arca OS 5.1 VMs, it also sometimes seem to hang during bios init and just stays like that until i kill the VM (host+R does nothing)

Also unbearably slow I/O with and without caching, idk if its an issue with this or upstream,
(EDIT: I would like to elaborate about the I/O being unbearably slow, the x86 version of System V from internet archive took 2 hours to install)

Patch used: https://github.com/cyberus-technology/virtualbox-kvm/releases/tag/dev-20240515
Compiled used an Unorthodoxed method using a Debian package installed under the VSO Vanilla OS container
(Used this monstrosity of a patch: **https://github.com/fluoriteByte/VirtualBox-KVM-patch-for-debian-sources**)

**Comment by tpressure** [comment]
@fluoriteByte thanks for the report. I doubt that this is related to running on AMD. I'd rather think this is an configuration or missing feature issue.

Can you please open a new issue to track this and upload your VM configuration and a log file from a VM start and shutdown cycle?

**Comment by amd1890** [comment]
i am also an amd user and am able to use this without error and often

**Comment by RodoMa92** [comment]
Same here on a 5900x, no issues detected on my end. This finally enables me to also have KVM VMs without having it clashing with them. Thanks a lot for your work :)

**Comment by sandikata** [comment]
> Same here on a 5900x, no issues detected on my end. This finally enables me to have also KVM VMs without having it clashing with them. Thanks a lot for your work :)

But no network support at all. 😁

**Comment by tpressure** [comment]
> > Same here on a 5900x, no issues detected on my end. This finally enables me to have also KVM VMs without having it clashing with them. Thanks a lot for your work :)
> 
> But no network support at all. 😁

> Readme: Only regular NAT is easily supported. More complex setups will need manual configuration, e.g., using tun/tap devices.

You can do more complex network setups by using tun/tap devices. You can't do this via the GUI though.
